booting_windows/readme.md

```diff
@@ -0,0 +1,76 @@
+# Running Windows over u-root
+
+## Prerequisites
+**WARNING**: the scripts will require sudo priviliges. It is highly advised that
+you read them carefully before running them!
+
+1. Golang, see `https://golang.org/`.
+1. Packages required to build Linux kernel
+1. A functional Windows Server 2019 image, asumed to exist at
+   `"${WORKSPACE}"/win2019.raw`. `setup.sh` will create a masking image over it,
+   so the original image will not be modified.
+1. An environment variable `EFI_WORKSPACE`, where files will be downloaded to or
+   otherwise created.
+
+## Installing the Modified u-root
+1.  Install u-root:
+
+    ```shell
+    go get github.com/u-root/u-root
+    ```
+
+1.  Change the uroot github remote to our modified one:
+
+    ```shell
+    pushd ~/go/src/github.com/u-root/u-root
+    git remote add oweisse https://github.com/oweisse/u-root  # our revised uroot repo
+    git fetch oweisse
+    git checkout -b kexec_test oweisse/kexec
```

Comment:
The upstream branch name should be kexec_test, not plain kexec. I.e., the command should be: 
git checkout -b kexec_test oweisse/kexec_test

Reply:
Yep, hence the `pushd ~/go/src/github.com/u-root/u-root` before hand. Let me know if you think this is not good enough.

Reply:
Fixed. See b28937d6